fundraisee/urls.py

```diff
@@ -15,9 +15,13 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from rest_framework_swagger.views import get_swagger_view
+
+schema_view = get_swagger_view(title='Fundraisee API')
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('api/', include('rest_framework.urls')),
+    path('', include('rest_framework.urls')),
     path('api/user/', include('accounts.urls')),
+    path('', schema_view),

```

Comment:
Why path regex in line 26 is same as line 24? Is it conflict?

Reply:
Line 24 to allow basic session auth : login,logout from rest framework urls

Ref
https://github.com/encode/django-rest-framework/blob/master/rest_framework/urls.py

Line 26 to allow swagger ui.

Shouldnt conflict because line 24 is adding login & logout urls only